CHARTER.md

```diff
@@ -1,59 +1,61 @@
 
 
-## Decision Proposals
+## Proposals
 
-Decisions can be started *only asychronously* in the [ART repository pull requests](https://github.com/arcalot/arcalot-round-table/pulls) via submitting a pull request with a new file to [`art-decisions/proposals`](art-decisions/proposals). You can use the included [template file](art-decisions/proposals/template.md) as a guideline. Decision proposals can only be submitted by ART members. If you would like to encourage discussion around a decision before any voting starts, please submit a draft PR instead and mark it ready for review once you are comfortable that you will not add any changes to your proposal.
+Proposals may only be made asychronously via the [ART repository pull requests](https://github.com/arcalot/arcalot-round-table/pulls) by submitting a pull request with a new file to [`art-decisions/proposals`](art-decisions/proposals). The provided [template file](art-decisions/proposals/template.md) may be used as a guideline for new proposals. Proposals may only be submitted by current ART members. If you would like to encourage discussion around a proposal before any voting starts, please submit a draft PR instead and mark it ready for review once you are comfortable that you will not make any changes to your proposal.
 
 ## Voting
 
-Each member of the Arcalot Round Table is eligible for voting if they have been a member of the ART for at least 30 days. For any decision that impacts the whole project or processes, a vote is started where a minimum of 4 votes is required with 75% approval majority among the voters.
+Each member of the ART is eligible for voting if they have been a member for at least 30 days. For any proposal that impacts the whole project or processes, **approval requires a minimum of four votes with 75% approval among those voting**.
 
-If the majority cannot be reached, the proposal is considered rejected and can be brought to the ART again 1 month after the rejection at the earliest. If the same decision has been rejected twice, it cannot be brought to the ART again.
+Any vote must be open for at least 7 calendar days to account for different time zones and other obligations and should last no longer than 30 days. The voting period should be stated in the proposal in calendar days. The voting period begins on the day the pull request is opened for review and ends at midnight [AoE](https://en.wikipedia.org/wiki/Anywhere_on_Earth) on the last day of the voting period. Proposals can be extended for no longer than 30 days total. Extensions can be made before the end of the current voting period by the original submitter, their direct representative, or one of the ART chairs by updating the PR with a note stating the number of calendar days for the extension. **If approval is not reached within the voting period deadline, the proposal is rejected.**
```

Comment:
"must" vs "should" ... we're placing a firm ("must") lower limit and an optional or conditional ("should") upper limit. I don't think that's really what we want. Maybe "must" be no longer than 30 days? Also, "be open" vs "last no longer" is a mismatch, so "must be open no longer than 30 days"?

"AoE" is odd usage, and somewhat confusing for a global organization. Essentially this could compel record keeping in any local timezone, and everyone participating needs to be aware of of the time zone of each proposal. How about just declaring UTC?

Reply:
AoE refers to the effective global end of a specific calendar day, which seems more "fair" than setting the deadline to midnight UTC. AoE specifically avoids being aware of the timezone of a proposal. We could maybe do midnight UTC-12:00, but technically I think AoE is more accurate.

Reply:
Sure, everyone in the world has the entire day of "July 16th" to respond: but ultimately it still refers to a *specific* timezone, but simply chooses one more obscure. This *really* just means "UTC-12:00", which is no more "objective" or "fair" than "UTC", unless "fair" means it's likely going to be mostly meaningless to almost everyone. 😆

Reply:
See also my comment below, but the gist of it is, AoE is fine if we're talking about _dates_, but we reference "midnight" which means that UTC is a far better choice.  And, as for being fair to everyone, the "clock" starts when the proposal is posted, which is "unfair" if the sun has already set for someone else; and, as proposed, the "clock" stops when the sun sets for someone _else_ which is also "unfair".  On the other hand, I don't see how using UTC is any less "fair" than using AoE, and, unless we're talking about _dates_ (and willing to let the deadline seem different for different timezones, unless the resident individuals know how to convert to UTC-12 (and, why is it _minus_ and not **plus**??)), UTC seems better suited to the problem because we all already know how to deal with it.